robot_framework/subprocess/masseoprettelse_mail.py

```diff
@@ -0,0 +1,172 @@
+"""This subprocess concerns mail functionality of the robot."""
+import json
+import re
+
+from OpenOrchestrator.orchestrator_connection.connection import OrchestratorConnection
+from itk_dev_shared_components.graph.authentication import GraphAccess
+from itk_dev_shared_components.graph import mail as graph_mail
+from itk_dev_shared_components.graph.mail import Email
+from itk_dev_shared_components.smtp import smtp_util
+
+from robot_framework import soup_mail
+from robot_framework import config
+
+
+def create_queue_from_emails(orchestrator_connection: OrchestratorConnection, graph_access: GraphAccess):
+    """Create a queue by reading emails and delete the emails after.
+
+    Args:
+        orchestrator_connection: A way to access the orchestrator to create the queue elements
+        graph_access: A token to access emails
+    """
+    # Check mailbox for emails to process
+    emails = _get_emails(graph_access)
+    emails.reverse()
+
+    # Parse each mail and add data to KMD Nova
+    for email in emails:
+        # Get data from email
+        data_dict = _parse_mail_text(email.body)
+        user_az = _get_az_from_email(data_dict["Bruger"])
+        user_email = _get_recipient_from_email(data_dict["Bruger"])
+        is_user_recognized = _check_az(orchestrator_connection, user_az)
+        _send_status_email(user_email, is_user_recognized, data_dict["Sagsoverskrift"])
```

Comment:
Probably best to not send the receipt before the queue elements are actually made, maybe?

Reply:
False! All we are saying in the status mail is whether or not the process has started. The mail is mostly to make sure we catch case workers not registered for operating the robot.